src/ralph/access_cards/models.py

```diff
@@ -25,7 +25,8 @@ class AccessCardStatus(Choices):
     damaged = _('damaged')
     used = _('in use')
     free = _('free')
-    return_in_progress = _('return in progres')
+    reserved = _("reserved")
+    return_in_progress = _('return in progress')
```

Comment:
Always add new statuses to the end of the list.

Reply:
ok. i will change it.